python/cudf/io/parquet.py

```diff
@@ -24,6 +24,19 @@ def _wrap_string(text):
         return ffi.new("char[]", text.encode())
 
 
+@ioutils.doc_read_parquet_metadata()
+def read_parquet_metadata(path):
+    """{docstring}"""
+
+    pq_file = pq.ParquetFile(path)
+
+    num_rows = pq_file.metadata.num_rows
+    num_row_groups = pq_file.num_row_groups
+    col_names = pq_file.schema.names
+
+    return num_rows, num_row_groups, col_names
```

Comment:
We may want to consider moving this to a libcudf API in the future so that the C++, Python, and other bindings implementations are aligned if we ever move away from using Arrow for this info.

Reply:
Yup, good point. In the future, we would probably expose a Parquet Reader class, and possible a ParquetMetadata/Schema class separately. Then we could optionally pass the latter to the former for actual dataset reading.

Reply:
I believe this will be necessary for BlazingSQL now anyway right @felipeblazing/@williamBlazing